[ADD] mailing: port 25 blocked for SMTP (#689)
odoo/documentation-user: discuss/email_servers.rst

For example, for "foo.com" they should have a record "odoo._domainkey.foo.com" 
that is a CNAME with the value "odoo._domainkey.odoo.com".

Restriction
-----------

> Please note that the port 25 will remain blocked for security reason. Try using 587, 465 or 2525.

Reviewer: ```suggestion
Please note that the port 25 is blocked for security reasons. Try using 587, 465 or 2525.
```
